tests/run-tests.sh

```diff
@@ -3,18 +3,50 @@
 set -euvo pipefail
 
 # This script assumes that it is running from the root of the Wit workspace.
-
 api_scala_sifive_path=./api-scala-sifive
+tests_path=$api_scala_sifive_path/tests
+coursier=./scala/coursier
+
+# Install jq if it is not already (as in environment-blockci-sifive)
+if hash jq 2>/dev/null; then
+  # jq already installed
+  jq=$(which jq)
+else
+  jq_linux_url=https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
```

Comment:
We could also fail out with a message out if they don't have `jq`

Reply:
I added this because environment-blockci doesn't have it so this is specifically downloading it for that case 🙈 @richardxia kind of blessed this

Reply:
Just for the record, I blessed the downloading of the binary, but I think I probably would have suggested the  separated it out into a separate setup script/step so that we don't couple the two together. I could have `jq` installed locally but just not on my PATH or something, and I may want to be reminded to set up my PATH to have it there.